examples/js/archive.js

```diff
@@ -383,6 +359,60 @@ shareMapBtn.addEventListener('click', () => {
   bootstrap.Modal.getInstance(shareModal).show()
 });
 
+// aggregate coordinates of all images into an array
+function getCornerBounds(imgCollection) {
+  let cornerBounds = []; 
+
+  // aggregate coordinates for multiple images int cornerBounds
+  if (imgCollection.length > 1) { 
+    imgCollection.forEach((imageObj) => {
+      if (imageObj.nodes) {
+        for(let i = 0; i < imageObj.nodes.length; i++) {
+          let corner = [];
+          corner[0] = imageObj.nodes[i].lat;
+          corner[1] = imageObj.nodes[i].lon;
+          cornerBounds.push(corner); // then we have array of arrays e.g., [ [lat, lon], [..], [..], [], [], [], [], [] ] for two images etc...
+        }        
+      }
+    });
+  } else { // aggregate coordinates for a single image into cornerBounds
+    if (imgCollection[0].nodes) {
+      for(let i = 0; i < imgCollection[0].nodes.length; i++) {
+        let corner = [];
+        corner[0] = imgCollection[0].nodes[i].lat;
+        corner[1] = imgCollection[0].nodes[i].lon;
+        cornerBounds.push(corner); // then we have [ [lat, long], [..], [..], [..] ] for just one image...
+      } 
+    }
+  }
+
+  return cornerBounds;
+}
+
+// converts legacy json objects (from mapknitter.org) to working format
+function transform(imgObj) {
+  let transformedImgObj = {};
+  
+  transformedImgObj.avg_cm_per_pixel = '';
+  transformedImgObj.collection = [];
+
+  imgObj.map((imgObj) => {    
+    if (imgObj.nodes.length) {
+      let tempNodes = [];
+
+      for(let i = 0; i < imgObj.nodes.length; i++) {
+        tempNodes.push({lat: imgObj.nodes[i].lat, lon: imgObj.nodes[i].lon});
+      }
+      imgObj.nodes = tempNodes; // overwrites the existing "nodes" key which points to a richer array. Access "imgObj.nodes" before this assignment to see the riicher array
```

Comment:
We could also assign the old one to a property like `oldNodes` or `legacyNodes` if you think people will want to access it. But it seems like we can discard it because we're not overwriting anything irrecoverable (from the original document) or losing any information in the new format, right?

Reply:
```suggestion
      imgObj.nodes = tempNodes; // overwrites the existing "nodes" key which points to a richer array. Access "imgObj.nodes" before this assignment to see the richer array
```

Reply:
> We could also assign the old one to a property like `oldNodes` or `legacyNodes` if you think people will want to access it. But it seems like we can discard it because we're not overwriting anything irrecoverable (from the original document) or losing any information in the new format, right?

We will not lose any information. Reading `imgObj.nodes` before line #406 gives complete access to the legacy node to anyone who needs it.